File: time.java

import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;
import java.util.StringTokenizer;

class l_kul {
	
	static long mul[]=new long[10001];
	public static void main(String s[]) throws IOException
	{
		BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
		int tcases = Integer.parseInt(br.readLine());
		boolean arr[]=new boolean[1441];
		while(tcases--!=0)
		{
			StringTokenizer st = new StringTokenizer(br.readLine(), ":-");
			int a=Integer.parseInt(st.nextToken())*60+Integer.parseInt(st.nextToken());
			int b=Integer.parseInt(st.nextToken())*60+Integer.parseInt(st.nextToken());
			for(int i=a;i<b;i++)
				if(arr[i])
				{
					System.out.println("Will need a moderator!");
					System.exit(0);
				}
				else
					arr[i]=true;
		}
		System.out.println("Who needs a moderator?");
	}
	
	
	
}

File: time.c
#include<stdio.h>
int main()
{
int a1[100],a2[100],b1[100],b2[100];
int t,i,j;
char s[12];
scanf("%d",&t);
for(i=0;i<t;i++)
{
scanf("%s",s);
a1[i]=(s[0]-'0')*10+(s[1]-'0');
a2[i]=(s[3]-'0')*10+(s[4]-'0');
b1[i]=(s[6]-'0')*10+(s[7]-'0');
b2[i]=(s[9]-'0')*10+(s[10]-'0');
}
i=0;
for(j=1;j<t;j++)
{
if(a1[j]>=b1[i]&&a2[j]>=b2[i])
{
printf("Who needs a moderator?\n");
i=j;
}
else
{
printf( "Will need a moderator!");
}
}
return 0;
}